.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: [3.5, 3.8]
```

We can drop `python 3.5` version as well and update the dependencies accordingly.
```suggestion
        python-version: [3.8]
```